src/backend/adapters/angular2.ts

```diff
@@ -24,7 +24,7 @@
  * Supports up to 2.0.0-beta-1
  */
 
-declare var ng: { probe: Function };
+declare var ng: { probe: Function, coreTokens: any };
```

Comment:
Good call...I need to double check but for some reason I seem to recall that it was typed but the typing weren't exported. Probably has changed since this is a public API.